### PR TITLE
FEAT: Event time filter

### LINKED ISSUE #3140: Event time filter
Hello,

I would like to be able to have the date picker also have a time option. For example, I can just view events that are overnight as there are so many during the day.

### PR DESCRIPTION
**Pull request by NickM-27**
Implementation of #3140

### CODE REVIEW

**Comment by NickM-27** [comment]
A couple screenshots showing this in action

Screen Shot 2022-06-05 at 14 36 07 PM

Screen Shot 2022-06-05 at 14 36 30 PM

**Comment by NickM-27** [comment]
Starts off as a time icon next to the calendar

Screen Shot 2022-06-05 at 14 53 42 PM

File: frigate/http.py [hunk 1]
```diff
@@ -539,6 +540,16 @@ def events():
...
 
+    if time_range != DEFAULT_TIME_RANGE:
+        times = time_range.split(",")
+        time_before = times[0]
+        time_after = times[1]
+
+        start_hour_fun = fn.strftime(
+            "%H:%M", fn.datetime(Event.start_time, "unixepoch", "localtime")
+        )
+        clauses.append((start_hour_fun < time_before and start_hour_fun > time_after))
```
Comment (NickM-27) [inline frigate/http.py:551]: I tried using `clauses.append((time_after < start_hour_fun < time_before))` but it didn't respect `time_after <`

Comment (NickM-27) [reply frigate/http.py]: Actually further testing and this didn't work

**Comment by sinamics** [comment]
@NickM-27 are you planning to complete this feature?
I started to add this functionality, but realized now that you already have a pr for this.
This is were im currently at:

image

This is quite different approach than yours. Let me know if this is something i should continue on.

**Comment by NickM-27** [comment]
@sinamics feel free to continue with this

**Pull request closed without merging**